resources/uk/gov/hmcts/gradle/init.gradle

```
@@ -38,11 +38,21 @@ settingsEvaluated { settings ->
   }
 }
 
-// Enable JSON dependency check reports for monitoring of CVEs and suppressions.
 allprojects {
   afterEvaluate { project ->
     if (project.hasProperty('dependencyCheck')) {
+      // Any CVE should fail the build.
+      project.dependencyCheck.failBuildOnCVSS = 0
```

can't this all be in a closure? untested

```suggestion
      project {
        dependencyCheck {
          failBuildOnCVSS = 0
       	  formats += json
       	}
      }
```

---

👍